app/views/partners/profiles/edit/_partner_settings.html.erb

```diff
@@ -7,23 +7,21 @@
             <h3 class="card-title"><strong>Settings</strong></h3>
           </div>
           <div class="card-body">
+            <% if form.object.organization.enable_quantity_based_requests? %>
+              <%= form.check_box :enable_quantity_based_requests, as: :boolean %>&nbsp;
+              <%= form.label :enable_quantity_based_requests, "Enable Quantity-based Requests" %>
+            <% end %>
+            <br>
```

Comment:
I've also re-ordered these so that the edit form order matches the show page. That page looks like this:

<img width="766" alt="screenshot from show page" src="https://github.com/rubyforgood/human-essentials/assets/1507328/36109da8-19b8-41b3-89e4-16ab7ec2b7c4">